samples/net/sockets/coap_server/src/certificate.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
Is the copyright correct here?

Reply:
The file is copied from another example with that copyright notice.